core/src/main/scala/org/apache/spark/memory/MemoryManager.scala

```diff
@@ -30,7 +36,10 @@ import org.apache.spark.storage.{BlockId, BlockStatus, MemoryStore}
  * sorts and aggregations, while storage memory refers to that used for caching and propagating
  * internal data across the cluster. There exists one of these per JVM.
```

Comment:
we probably need to update this java doc to include all the new things that this class now does